test/contract/AllowListManager.spec.ts

```diff
@@ -0,0 +1,121 @@
+/// This file does not represent extensive unit tests, but rather just demonstrates an example
```

Comment:
Does it mean that `AllowListOffChainManaged` is intended just for testing and not for use as it is right now?
A test I'd like to see is that it fails on a bad signature.